### ISSUE TITLE
Return events on sendTransaction

### ISSUE THREAD

**Issue by naumenkogs** [ethereum/web3.js#1377]
Is there a way to obtain generated events on sendTransaction call (in the same way I get it from contract.methods.method_name.send())?

This could be useful to observe events on deferred transactions (since I can't use regular send() on contract in this case) or when transfer goes through a fallback payable function (for example token minting generates events)?

That's what I'm getting now as a result from sendTransaction:

```
{ transactionHash: '0xee0a4aa8aba9f8ceae4ba21e6d73190b093ee6671bcb1992cff1aeefd414c6e2',
  transactionIndex: 0,
  blockHash: '0xc956e5b6b87e255d462ad8a49619051f2227a97c486ad3e0fc52b119f8f46d60',
  blockNumber: 346,
  gasUsed: 99616,
  cumulativeGasUsed: 99616,
  contractAddress: null,
  logs:
   [ { logIndex: 0,
       transactionIndex: 0,
       transactionHash: '0xee0a4aa8aba9f8ceae4ba21e6d73190b093ee6671bcb1992cff1aeefd414c6e2',
       blockHash: '0xc956e5b6b87e255d462ad8a49619051f2227a97c486ad3e0fc52b119f8f46d60',
       blockNumber: 346,
       address: '0xa9FBC415b2969f3B65F2021F76F422ba82c9A062',
       data: '0x0000000000000000000000000000000000000000000000000000000000000001',
       topics: [Array],
       type: 'mined',
       id: 'log_eca044e4' },
     { logIndex: 1,
       transactionIndex: 0,
       transactionHash: '0xee0a4aa8aba9f8ceae4ba21e6d73190b093ee6671bcb1992cff1aeefd414c6e2',
       blockHash: '0xc956e5b6b87e255d462ad8a49619051f2227a97c486ad3e0fc52b119f8f46d60',
       blockNumber: 346,
       address: '0xa9FBC415b2969f3B65F2021F76F422ba82c9A062',
       data: '0x00000000000000000000000000000000000000000000003635c9adc5dea00000',
       topics: [Array],
       type: 'mined',
       id: 'log_c18ea743' },
     { logIndex: 2,
       transactionIndex: 0,
       transactionHash: '0xee0a4aa8aba9f8ceae4ba21e6d73190b093ee6671bcb1992cff1aeefd414c6e2',
       blockHash: '0xc956e5b6b87e255d462ad8a49619051f2227a97c486ad3e0fc52b119f8f46d60',
       blockNumber: 346,
       address: '0x83BaB5D9066F354daA49a0C435b1bCD8AD5f1462',
       data: '0x0000000000000000000000000000000000000000000000000de0b6b3a764000000000000000000000000000000000000000000000000003635c9adc5dea00000',
       topics: [Array],
       type: 'mined',
       id: 'log_ba247c28' } ],
  status: 1 }
```

**Comment by nivida** [comment]
This should be implemented when the PR #2000 is merged and released.

**Comment by nivida** [comment]
Sorry I didn't see that you were meaning the eth.sendTransaction. Sadly, this isn't possible. Just use the contract methods if you would like to have these events decoded.
